target/hwaccess/hwAccessIntf.C

```diff
@@ -0,0 +1,84 @@
+#include <hwAccessIntf.H>
+#include "hwBaseAccess.H"
+#include "hwDirectAccess.H"
+#include "hwSbeAccess.H"
+
+namespace hwaccess
+{
+int HwAccessIntf::getCfamRegisters(TargetPtr_t target, uint32_t addr, uint32_t& val)
```

Comment:
This method gets a single register value, so better to name it getCfamRegister. (remove the s) and similarly for all the other methods